pages/war.php

```diff
@@ -117,11 +117,12 @@ function processWarBattleEnd($battle, $player): string {
         $battle_result .= "You have been defeated.";
         $player->health = 5;
         $player->ai_losses++;
-        $player->moveToVillage();
+        //$player->moveToVillage();
         $player->battle_id = 0;
         $player->last_pvp_ms = System::currentTimeMs();
+        $battle_result .= $warManager->handlePatrolWin($patrol_id);
```

Comment:
This function is slightly ambiguous to whether the player won or the NPCs did, we may want to name it a little more specifically

Reply:
`handleLossAgainstPatrol()` or similar